README.md

```diff
@@ -1,4 +1,5 @@
 Bem vindo à minha primeira atividade partindo do zero com classes de C++.
 Essa atividade consiste num jogo onde primeiramente determina-se um número limite, e em seguida, determinados jogadores tentam alcançar esse número com dados sem ultrapassa-lo.
 O jogador soma os dados até perder(ultrapassando o limite) ou caso queira parar.
-Vence o jogador que não ultrapassar e chegar mais perto do limite.
\ No newline at end of file
+Vence o jogador que não ultrapassar e chegar mais perto do limite.
```

Comment:
Pelo PR fica fácil achar o que mudou e comentar linha-a-linha e sugerir o que for preciso.